components/02-elements/minicart-content/_minicart-content.scss

```diff
@@ -0,0 +1,117 @@
+$minicart-content-margin      : 15px !default;
```

Comment:
Can we move this stuff to `modules`? Will be nice to at least keep the rule of not inheriting elements in elements, then we can remove it from modules.

Reply:
Comment to this specific line - it's not just a `margin`, more like a `spacing`, b/c it's in use in many cases.

Reply:
`minicart-content` moved to the modules BUT. When we are keeping the rule of elements and modules. Popup element should be in modules too. Than we can't keep minicart-content here b/c it's using on header in popup. And header can't use popup b/c it's module too 🤔

Reply:
I'd suggest changing the rules a little bit and make an inheritance of modules "legal", but the question is if it's technically possible and will not lead to some weird scenarios.

Reply:
which weird scenarios do you mean? I moved popup to modules too and seems to working fine in header

Reply:
tested on 2 same popups in header and working fine